lib/util/query_params.dart

```diff
@@ -0,0 +1,20 @@
+import 'dart:html';
```

Comment:
Out of scope for this PR, but it would be good to get the rest of the query parameters in here as well.

Reply:
You're right, created an issue to track: https://github.com/dart-lang/dart-pad/issues/1664